src/containers/Feed/feeds.jsx

```diff
@@ -0,0 +1,107 @@
+/* eslint-disable no-plusplus */
+import React, { Component } from 'react';
+import moment from 'moment';
+import { Select } from 'antd';
+import { Link } from 'react-router-dom';
+
+import NavBar from '../../components/navigationBar';
+import JournalCard from '../../components/JournalCard';
+import { months, fakeData } from './data';
+import LogoHeader from '../../components/LogoHeader';
+import './feeds.css';
+
+const { Option } = Select;
+class Feed extends Component {
+  state = {
+    data: [],
+    monthCount: months,
+  };
+
+  componentDidMount() {
+    // fetch all data from firebase firstore
+    // async
+    // const result = await axios.get(`https:-------`);
+    // const cards = result.data.data[0];
+    // cards === fackData
+    //
+    const monthArray = fakeData.map(journal =>
+      moment(journal.timestamp).format('MMMM')
+    );
+    const filteredObject = monthArray.reduce((acc, curr) => {
+      if (typeof acc[curr] == 'undefined') {
+        acc[curr] = 1;
+      } else {
+        acc[curr] += 1;
+      }
+      return acc;
+    }, {});
+
+    const keys = Object.keys(filteredObject);
+
+    for (let i = 0; i < months.length; i++) {
+      for (let j = 0; j < keys.length; j++) {
+        if (months[i].month === keys[j]) {
+          months[i].count = filteredObject[keys[j]];
+        }
+      }
+    }
+    return this.setState({ monthCount: months });
```

Comment:
@AlaaTaima I think you don't need `return` here.

Reply:
Yea, you are right , thanks for your comment .